### DISCUSSION TITLE
"Customize your notification experience" ever time I open the app

### DISCUSSION THREAD

**Discussion by chylex** [community/community#137157] [Mobile] [answered]
Every time I open the GitHub app, I see this...

![Screenshot_20240829-155105~2.png](https://github.com/user-attachments/assets/e2a07246-edd9-4755-b761-3a2790ab6469)

I already customized my settings years ago. Tapping either option hides it, only for it to come back the next time I open the app. I don't know how to get rid of this annoying prompt that takes up half the screen...

**Comment by stevepopovich** [comment]
Hi @chylex we are looking at this and will get this fixed very soon. Are you on the beta? Thanks for writing in!

**Reply by chylex** [reply]
Hi, it says beta on the version number, so I suppose so. Google Play says the GitHub app was updated yesterday; I just let it do its thing, I don't know where you can decide if you're on the beta branch.

I think the build number was 10210, but I can't go back to check if I remembered correctly because it will discard my comment. It would be useful if tapping the version number in the app copied it to clipboard.

**Reply by stevepopovich** [reply]
That's great. Thank you.

**Reply by stevepopovich** [reply, marked as answer]
We are releasing a fix for this. You should be able to update to it in the next 24 hours. Thanks for reporting!
